safe/metadata/exposure_layer_metadata.py

```diff
@@ -127,6 +127,13 @@ class ExposureLayerMetadata(GenericLayerMetadata):
             'inasafe/'
             'value_map/'
             'gco:Dictionary'),
+        'minimum_needs': (
```

Comment:
Where and when do we assign minimum_needs keyword in the exposure_layer_metadata? And why we put it here?